site/styles/_shift72.scss

```diff
@@ -133,6 +133,13 @@
   }
 }
 
+// some fields which are "readonly" shouldn't have the padding and font-size applied to them because they are text (not masked "****" values)
+.s72-readonly-text .s72-form-control-readonly {
+  font-size: unset;
+  padding: 0 0.875rem;
+  border-radius: 0.25rem !important;
```

Comment:
These values look like hacks.
Why not use the variables that exist for site wide [border-radius](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L654) and [padding](https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss#L630)?